PAC/common/bus_coupler_io.h

```diff
@@ -366,6 +366,7 @@ class io_manager
 			char    name[20];      ///< Имя.
 
 			bool is_active;          ///< Признак работающего узла.
+            bool io_error_flag;

```

Comment:
Предлагаю использовать современный подход для инициализации членов (прямо во время объявления):
```suggestion
            bool io_error_flag = false;

```